tests/modules/programs/hexchat/hexchat-basic-configuration.nix

```diff
@@ -0,0 +1,65 @@
+{ config, lib, pkgs, ... }:
+
+{
+  config = {
+    programs.hexchat = {
+      enable = true;
+      overwriteConfigFiles = true;
+      channels = {
+        freenode = {
+          charset = "UTF-8 (Unicode)";
+          userName = "user";
+          password = "password";
```

Comment:
Too bad the channel password has to be passed in cleartext in the configuration.

It is possible to specify the password somewhere else or have it in a separate file?

Reply:
I don't think so, but maybe @superherointj can confirm.

Reply:
I haven't reached the point of solving the clear text password problem. When I get there I might provide a better solution.

Reply:
Yeah, it can wait. Cleartext passwords have been a longstanding issue in NixOS.